examples/compose.js

```diff
@@ -0,0 +1,29 @@
+import dotty from 'dotty';
+import merge from 'lodash/object/merge';
+
+const getDescriptorProps = (descriptorName, composables) => {
+  return composables.map(composable => {
+    return dotty.get(composable, 'compose.' + descriptorName);
+  });
+};
+
+function compose (...composables) {
+  const composable = {};
```

Comment:
Yeah, these are just placeholders. Like I said, I haven't implemented the stamp bit yet. :)